### PR TITLE
php8 return types compatible

### PR DESCRIPTION
**Pull request by maikelohcfg**
Updated methods return types for php8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": ">=7.4",
+        "php": "^8.0",
```
Comment (Nek-) [inline composer.json:15]: This is not going to happen any soon.

**Review by Nek- — COMMENTED** [review]
Hello, thank you very much for your contribution but I don't think that we're not going to end the support of PHP 7.4 in the next 6 months.

To me, this PR is not required for anything but removing Symfony deprecation. It could probably be done by adding explicit comments instead.

**Comment by maikelohcfg** [comment]
Hello @Nek- thanks for your quick review and your opinion, based on that I have changed the native php 8 types on methods to annotation return types. 

You are right, this PR remove Symfony deprecations alerts. I like to keep my projects clean of deprecations alerts, so I'm guilty 😜

File: composer.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=7.4 || ^8.0",
```
Comment (maxhelias) [inline composer.json:15]: No need to change it here

**Review by maxhelias — COMMENTED** [review]
Thanks for this work

File: composer.json (hunk 2 above)
Comment (maikelohcfg) [reply composer.json]: Fixed, thanks for your review

Comment (Nek-) [reply composer.json]: Actually >=7.4 was fine, why changing it to this?

File: src/Serializer/Normalizer/PhoneNumberNormalizer.php [hunk 3]
```diff
@@ -82,11 +84,12 @@ public function supportsNormalization($data, $format = null)
      * {@inheritdoc}
      *
      * @throws UnexpectedValueException
+     * @return mixed
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
         if (null === $data) {
-            return;
+            return null;
```
Comment (Nek-) [inline src/Serializer/Normalizer/PhoneNumberNormalizer.php:92]: Please remove this change as well `return;` was fine and it follows Sf coding style

**Comment by Nek-** [comment]
It have far more changes to be merged like this hehe. Thank you very much @maikelohcfg .

File: src/Serializer/Normalizer/PhoneNumberNormalizer.php (hunk 3 above)
Comment (maikelohcfg) [reply src/Serializer/Normalizer/PhoneNumberNormalizer.php]: Hello done and pushed

**Comment by maikelohcfg** [comment]
> It have far more changes to be merged like this hehe. Thank you very much @maikelohcfg .

Community is not always about receive, we must contribute too. So thanks to you too.

**Review by Nek- — APPROVED** [review]

**Comment by maxhelias** [comment]
@maikelohcfg Just one last thing please 🙏 Can you look at why PHP-CS-Fixer is not happy ? 🤔

**Comment by maikelohcfg** [comment]
@maxhelias done with php-cs-fixer

**Review by Nek- — APPROVED** [review]

**Pull request merged**